build.ps1

```diff
@@ -0,0 +1,4 @@
+dotnet --info
```

Comment:
Switching to PowerShell scripts was not part of the plan for this PR (can be discussed/raised as a separate issue) so kindly revert to using `build.cmd`. `build.ps1` is adding no value at the moment because in the end it's just calling `dotnet.exe` to do all the work and Windows batch scripts to do that more naturally. In fact, it worsens the situation right now because in the PowerShell version, all 4 steps will execute ignoring any errors in a previous step.